activesupport/lib/active_support/railtie.rb

```diff
@@ -47,5 +47,11 @@ class Railtie < Rails::Railtie # :nodoc:
         ActiveSupport.send(k, v) if ActiveSupport.respond_to? k
       end
     end
+
+    initializer "active_support.set_claims_expiration" do |app|
+      require 'active_support/claims'
+      app.config.active_support.claims_expiration ||= 1.month
```

Comment:
I'm still not sure if we should expose Claims to general users (that's another question we should ask in the PR). Anyway, let's rename this to `messages_expires_in`.